plugins/generator-1.19.4/forge-1.19.4/triggers/entity_pre_hurt.java.ftl

```diff
@@ -12,9 +12,11 @@
 				"entity": "event.getEntity()",
 				"damagesource": "event.getSource()",
 				"sourceentity": "event.getSource().getEntity()",
+				"immediatesourceentity": "event.getSource().getDirectEntity()",
 				"event": "event"
 				}/>
 			</#compress></#assign>
 			execute(event<#if dependenciesCode?has_content>,</#if>${dependenciesCode});
 		}
-	}
\ No newline at end of file
+	}

```

Comment:
Those changes are unrelated. Please use proper IDE to do changes, not GitHub online editor.